### PR TITLE
[EASY-ROMAN-TO-INTEGER] Implemented and added tests for RomanToInteger

### PR DESCRIPTION
**Pull request by mavharsha**
Implemented and added tests for RomanToInteger.

### CODE REVIEW

File: src/main/java/com/mavharsha/leetcode/easy/RomanToInteger.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+     * Convert Roman number to Integer.
+     *
+     * @param input roman number as String
+     * @return the number corresponding to the roman number
+     */
+    public int convert(String input) {
+        int sum = 0;
+        char currentChar = ' ';
+
+        if (input == null || input.equalsIgnoreCase("")) {
```
Comment (odiljon) [inline src/main/java/com/mavharsha/leetcode/easy/RomanToInteger.java]: you can use `input.length() == 0` as well

Comment (mavharsha) [reply src/main/java/com/mavharsha/leetcode/easy/RomanToInteger.java]: Sure! Can be and will be changed to `length == 0`

File: src/main/java/com/mavharsha/leetcode/easy/RomanToInteger.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            return sum;
+        }
+        input = input.toUpperCase();
+        if (input.length() == 1) {
+            return romanToIntegerMap.containsKey(input) ? romanToIntegerMap.get(input) : sum;
+        }
+
+        for (int index = (input.length() - 1); index >= 0; index--) {
```
Comment (odiljon) [inline src/main/java/com/mavharsha/leetcode/easy/RomanToInteger.java]: why you are looping reverse order? 🤔

Comment (mavharsha) [reply src/main/java/com/mavharsha/leetcode/easy/RomanToInteger.java]: evaluating the string from right to left.

**Pull request merged**